src/Widgets/VideoWidget/VideoWidgetEditingConfig.ts

```diff
@@ -29,8 +29,12 @@ provideEditingConfig(VideoWidget, {
         'This poster image is shown, until the video is loaded.' +
         ' Without an poster image, the browser may show the first frame of the video.',
     },
+    noDownload: {
+      title: 'Hide download button?',
+      description: 'Only supported by some browsers.',
```

Comment:
* This simple wording has the potential to confuse editors. We should make clear that this is just a presentational hint and not all visitors will see the difference.
* The new option lets editors configure a very minor cosmetic detail, which in fact may reduce the usability of the video for some visitors.
* There appears to be back-pressure from browser vendors to support this option.

I'd recommend to not add this feature. If a project knows the limitations and is willing to accept them, a developer can easily add the respective option. The base implementation should stick to widely supported options.